src/main/convert.js

```diff
@@ -208,6 +208,19 @@ export class ThriftFileConverter {
     return def.definitions.map((d, index) => `"${d.id.name}"`).join(' | ');
   };
 
+  generateEnumMap = (def: Enum) => {
+    const header = '{';
+    const values = def.definitions
+      .map(
+        (d, index) => `  "${d.id.name}": ${d.value ? d.value.value : index},`
+      )
+      .join('\n');
+    const footer = '}';
+
+    const mapDefinition = [header, values, footer].join('\n');
+    return `export const ${def.id.name}ValueMap = ${mapDefinition};`;
```

Comment:
There's the possibility for a conflict if you add values like this. In this case, you might have a struct itself that has `ValueMap` in it, for example.